### DISCUSSION TITLE
How to get whitelisted on GitHub Spark?

### DISCUSSION THREAD

**Discussion by karlo-franic** [community/community#148014] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I was wondering what are the requirements to get on the whitelist for GitHub Spark?
Thanks!

**Comment by ebndev** [comment]
Hi @karlo-franic, thanks for being a part of the GitHub Community!

You can sign up for the waitlist for the technical preview [on our website](https://github.com/github_spark_waitlist_signup/join). 😄

**Reply by karlo-franic** [reply]
Yes I know. I signed up about a month and a half ago, that's why I'm asking what are the requirements. :)

**Comment by iamperii** [comment]
Thank you for your suggestion this helped me a lot, I will follow these especially that advice suggested by the Simpson...
